kqueen/auth/common.py

```diff
@@ -45,9 +45,10 @@ def generate_auth_options(auth_list):
     for m in methods:
         if m in AUTH_MODULES:
             auth_options[m] = AUTH_MODULES[m]
-
+        else:
+            logger.debug('Specified in config auth method {} is not found in the auth modules specification ')
     if not auth_options:
```

Comment:
seems need to refactor this, coz we already call ```             auth_options[m] = AUTH_MODULES[m]
``` on raw 47

Reply:
good point